azure-stack/hci/concepts/azure-stack-hci-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
```

Comment:
Change to: Monitor at scale using the Azure Stack HCI overview page (preview)

Reply:
Done > updated the title.